CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-beta.1 November 18, 2021
+
+## Features ✨ and improvements 🏁
+* Remove `MapView` argument when constructing `AnnotationManager`. ([#766](https://github.com/mapbox/mapbox-maps-android/pull/766))
+* Implement cluster API on top of `MapboxMap.queryFeatureExtensions`. ([#773](https://github.com/mapbox/mapbox-maps-android/pull/773))
+* Introduce view annotation support which allows adding Android views on top of the `MapView` anchored to geo-point. ([#834](https://github.com/mapbox/mapbox-maps-android/pull/834))
+* Add heatmap and circle layer support to globe view. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Improve performance for querying large number of features. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Cache layer layout key inside layer, so that it is not re-evaluated at every parse of the every tile. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Model layer now supports loading of external `gltf` files. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Add cancelable Query Rendered Features API to `MapboxMap`. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Core renderer prints its version on initialization. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Introduce experimental `MapboxMap.setMemoryBudget` method for setting memory budget for the map and runtime "resource-budget" property for data sources. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Improve performance by avoiding re-layout of invisible fading tiles. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Reduce memory consumption of a model layer by avoiding duplicate texture uploads. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
```

Comment:
same as above, we don't expose model layer publicly currently.

Reply:
👍